edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Core.Test/DeploymentConfigInfoTest.cs

```diff
@@ -3,8 +3,10 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Core.Test
 {
     using System.Collections.Generic;
     using Microsoft.Azure.Devices.Edge.Agent.Core.Test.ConfigSources;
+    using Microsoft.Azure.Devices.Edge.Util.Test.Common;
     using Xunit;
 
+    [Unit]
```

Comment:
So all these tests weren't running at all? Did you discover this because the code coverage tool flagged these files as untested?

Reply:
They weren't running in the gated builds because we're always careful to specify a filter. I guess they'd run if you just do a vanilla `dotnet test` at your desk.

I noticed one assembly was reporting 0% coverage even though it had tests. That led me to open up VS Test Explorer, group tests by category, and expand the "uncategorized" tree. :)

Reply:
I don't know of a way to flag uncategorized tests, but that would be nice...